zkevm-circuits/src/evm_circuit/execution/error_oog_log.rs

```diff
@@ -124,13 +126,13 @@ impl<F: Field> ExecutionGadget<F> for ErrorOOGLogGadget<F> {
         self.is_opcode_logn
             .assign(region, offset, F::from(topic_count), F::from(5u64))?;
 
+        let gas_cost = GasCost::LOG.as_u64()
```

Comment:
don't need to change for log. since Log only have dynamic gas const. the old one is OK.